### ISSUE TITLE
Support binding a key multiple times

### ISSUE THREAD

**Issue by mathieucaroff** [obv-mikhail/InputBot#10]
### Current behavior

Currently, when a key is bound multiple times, the last binding operation overrides the previous ones.

### Expected behavior

When a key or mouse event is bound multiple times, InputBot dispatches the event to all callback functions, in the order in which they were registered.

Note: this would be a breaking change.

After thinking about it, I found the current API, with `KeydbKey:unbind(&self)`, would make a rather coarse API, requiring to unregister all callbacks at once.

### Alternative

Provide an new interface which handles dispatch.

**Comment by obv-mikhail** [comment]
@mathieucaroff I agree that we would probably want to have a different unbind method. Maybe something like `KeydbKey:unbind(&self, fn)` where `fn` is the function bound to the event originally.

How would the alternative with an interface for handling dispatch look like?

**Comment by D01B0EB** [comment]
https://github.com/obv-mikhail/InputBot/issues/102#issuecomment-3992163204
